src/server/bg_services/semaphore_monitor.js

```diff
@@ -72,8 +72,8 @@ class SemaphoreMonitor {
         }
     }
 
-    sample_nsfs_semaphore() {
-        const buffers_pool_sem = buffers_pool.sem;
+    sample_nsfs_l_semaphore() {
```

Comment:
how hard it is to have all semaphores in the stats?
I see in line 92 we call:
```
endpoint_stats_collector.update_semaphore_state(semaphore_report, "nsfs", this.report_sample_sizes);
```
perhaps just cann this once per buffer_pool with different names?

Reply:
done, 4 new semaphores are now supported.